src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,220 @@
+    }
+
+    th:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+      padding-left: $spacing-lg;
+    }
+
+    th.#{$prefix}--table-column-icon:hover,
+    th.#{$prefix}--table-column-menu:hover,
+    th.#{$prefix}--table-expand-v2:hover {
+      background: $ui-03;
+    }
+
+    tbody tr td {
+      background: $ui-01;
+      border-bottom: 1px solid $ui-03;
+      border-top: 1px solid $ui-01;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2) {
+      padding: rem(14px) $spacing-sm;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2):first-of-type {
+      padding-left: $spacing-lg;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2):last-of-type {
+      padding-right: $spacing-lg;
+    }
+
+    .#{$prefix}--table-header-label {
+      display: block;
+      padding: rem(14px) 0;
+    }
+
+    //width for checkbox columns
+    .#{$prefix}--table-column-checkbox {
+      width: rem(50px);
+      min-width: rem(50px);
+      padding: rem(10px) $spacing-md 0 $spacing-md;
+    }
+
+    //width for overflow menu columns
+    .#{$prefix}--table-column-menu {
+      width: rem(52px);
+      min-width: rem(52px);
+      padding-top: rem(8px);
+      padding-right: rem(8px);
+    }
+
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu[aria-expanded='false']:hover {
+      background: $ui-03;
+    }
+
+    // Checbox and Overflow Menu Overrides
+    .#{$prefix}--checkbox-label {
+      padding-left: rem(28px);
+    }
+
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu[aria-expanded='false']:focus {
+      @include focus-outline('outline');
+    }
+
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu[aria-expanded='true']:focus {
+      outline: none;
+    }
+
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu .#{$prefix}--overflow-menu__icon {
+      opacity: 0;
+    }
+
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu:hover .#{$prefix}--overflow-menu__icon,
+    td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu:focus .#{$prefix}--overflow-menu__icon,
+    tr:hover td.#{$prefix}--table-column-menu .#{$prefix}--overflow-menu .#{$prefix}--overflow-menu__icon {
+      opacity: 1;
+    }
+  }
+
+  //-------------------------------------------------
+  //NUMERICAL COLUMN
+  //-------------------------------------------------
+  td.#{$prefix}--data-table-v2--numerical-cell,
+  th.#{$prefix}--data-table-v2--numerical-cell {
+    text-align: right;
+  }
+
+  //-------------------------------------------------
+  //DISABLED
+  //-------------------------------------------------
+  tbody tr.#{$prefix}--row-disabled {
```

Comment:
Are disabled rows a v9 feature?

Reply:
I don't think they're a feature in v9. There aren't any styles for it, but it's spec'd out for v10.

Reply:
Should we punt on this since we're going for v9 parity first then?

Reply:
Is this not considered part of default styles?

Reply:
It's more so that we are focusing on v9 parity before adding additional functionality in this release 👍